content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -348,6 +348,20 @@ $ dvc exp show --csv | csvstat
 
 [csvkit]: https://csvkit.readthedocs.io/en/latest/
 
+## Show the experiments table as a Parallel Coordinates plot
+
+`dvc exp show` can also generate an interactive Parallel Coordinates plot, with
+`--html`.
+
+The `--html` option can be combined with the other flags of `dvc exp show`. For
+example, the flag `--sort-by` would set the color of the lines.
+
```

Comment:
I think instead of "sets the color", we should be saying, "the intensity of the color is determined by the value of the column you select with `--sort-by`." The user is not selecting a color, they select a column to highlight the results.